### PR TITLE
Reinstate OSX on Travis

### PR DESCRIPTION
**Pull request by patricksnape**

### CODE REVIEW

**Comment by patricksnape** [comment]
@jabooth Happy?

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
 
 os:
-- linux
+  - osx
+  - linux
+osx_image: xcode6.4
 
 git:
-  depth: 200
+  depth: 1
```
Comment (jabooth) [inline .travis.yml]: Why change this, didn't we have issues with insufficient depth in past?

File: .travis.yml [hunk 2]
```diff
@@ -22,14 +30,4 @@ script:
 - ~/miniconda/bin/python condaci.py build ./conda
 
 notifications:
-  slack:
```
Comment (jabooth) [inline .travis.yml]: Any reason we are disabled slack build notifications too (or is this how handled though a proper integration? Don't remember)

File: .travis.yml (hunk 1 above)
Comment (patricksnape) [inline .travis.yml]: I don't believe so - was just cleaning up the YAML a bit

**Comment by patricksnape** [comment]
OK I went back to minimal changes

**Comment by jabooth** [comment]
Cool, less risk of issues. I'm happy merge whenever you're happy CI is passing

**Pull request merged**